frontend/awx/access/credentials/CredentialForm.tsx

```diff
@@ -518,6 +601,22 @@ function CredentialSubForm({
               />
             );
           } else {
+            if (isEncryptedValues) {
```

Comment:
This if statement is unnecessary - the next line checks that the field in question has a value of `$encrypted$`